tests/testthat/test-int-latent_individual.R

```diff
@@ -85,6 +85,43 @@ test_that("epidist.epidist_latent_individual Stan code has no syntax errors and
   expect_no_error(mod_gamma$compile())
 })
 
+test_that("epidist.epidist_latent_individual fits and the MCMC converges in the gamma delay case", { # nolint: line_length_linter.
+  # Note: this test is stochastic. See note at the top of this script
+  skip_on_cran()
+  set.seed(1)
+  fit_gamma <- epidist(
```

Comment:
We aren't passing a seed to brms?

Reply:
Good spot, if it's not there then it should be and I'll add it